spec/jobs/batch_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'spec_helper'
+require 'spec/support/persistence'
+require 'spec/support/events'
+require 'spec/support/dummy'
+require 'spec/support/env'
+require 'routemaster/jobs/batch'
+require 'routemaster/services/codec'
+# require 'routemaster/services/deliver'
+
+module Routemaster
+  describe Jobs::Batch do
+    subject { described_class.new(delivery: delivery) }
+
+    let(:subscriber) {
+      Routemaster::Models::Subscriber.new(name: 'alice').tap do |s|
+        s.timeout = 0
+      end
+    }
+    let(:messages) { [make_event, make_event] }
+    let(:delivery) { double 'delivery', call:nil }
```

Comment:
Actually, this one is a stand-in for the `Services::Deliver` _class_, so `instance_double` wouldn't help? (it'd be an instance of `Class`?)